### ISSUE TITLE
syncnet update weights

### ISSUE THREAD

**Issue by cranberrymuffin** [cranberrymuffin/anime-dub#13]
update the weights using optimizer and contrastive loss function from issue #12

**Comment by cranberrymuffin** [comment]
fixed using https://medium.com/predict/face-recognition-from-scratch-using-siamese-networks-and-tensorflow-df03e32f8cd0 as reference
